tests/misc.sh

```diff
@@ -258,10 +245,11 @@ supported()
 
 require()
 {
-	if supported ${1}; then
-		return
+	if ! supported ${1}; then
+		echo "1..1"
```

Comment:
This should be `0..1`: https://testanything.org/tap-specification.html .

This doesn't cause grief with prove, but it does with things like kyua which are more strict with parsing TAP.

Reply:
Dang dyslexia: `1..0`.

Reply:
Thanks, changed.